object-cache.php

```diff
@@ -414,6 +440,12 @@ function get_blog_flush_number() {
 		return $this->flush_number[ $this->blog_prefix ];
 	}
 
+	function local_flush() {
+		$this->cache = array();
+
+		return true;
+	}
```

Comment:
For consistency with the new drop-in, could we rename this to `flush_runtime()` so we don't need to worry about dropping a public method later?

And then I think we should also clear out group_ops when this is called. Clearing local cache is usually for long-running tasks to conserve memory, and group_ops is the other thing that keeps growing.

https://github.com/Automattic/wp-cache-memcached/blob/edf499e835db57ff5adfb79190886d5e94ea300b/includes/wp-object-cache.php#L669-L674

Reply:
Thanks for the review @WPprodigy . Actioned the feedback.